src/bin/travis/testCentaurJes.sh

```diff
@@ -13,7 +13,7 @@ fi
 
 printTravisHeartbeat() {
     # Sleep one minute between printouts, but don't zombie for more than two hours
-    for ((i=0; i < 120; i++)); do
+    for ((i=0; i < 180; i++)); do
```

Comment:
Is this strictly necessary?  If so could you please update comment to reflect "three" hours

Reply:
The Joint Discovery workflow takes ~2 hours and 30min to complete. Both the TravisHearbeat and MaxworkflowLength need to be set to 3 hours or less, else travis will stop with time limit error.

Reply:
ToL (not sure on feasibility) it would be nice if all of these were fed from the same number so we only needed to update one place.